src/debian/11/helix/amd64/Dockerfile

```
@@ -48,7 +48,7 @@ RUN curl -LO https://packages.microsoft.com/keys/microsoft.asc && \
     rm microsoft.asc && \
     apt-add-repository https://packages.microsoft.com/debian/11/prod && \
     apt-get update && \
-    apt-get install -y libmsquic && \
+    apt-get install -y libnuma1 libmsquic && \
```

Nit: I understand only two packages are here but might as well follow best practices right off the bat and have them listed alphabetically. My preference is to have them listed on separate lines as well. Same applies to other Dockerfiles.

```suggestion
    apt-get install -y \
      libmsquic \
      libnuma1 && \
```

---

I can do that. this will hopefully be reversed soon as msquic fixes their packages e.g. this should not really be needed.